Casks/blender-benchmark.rb

```diff
@@ -1,15 +1,20 @@
 cask "blender-benchmark" do
-  version "2.0.4"
-  sha256 "5197bf7d262bda630ea2c01bf0d097c2f7f1533f1affe33d0ceac383d3c30b2e"
+  version "3.0.0,2.0"
+  sha256 "d6fd4dd6f203331661a6139a9c0d00806dda750d9aefd13670d87d4336204d08"
 
-  url "https://opendata.blender.org/cdn/BlenderBenchmark#{version.major_minor}/launcher/benchmark-launcher-#{version}-macos.dmg"
+  url "https://opendata.blender.org/cdn/BlenderBenchmark#{version.csv.second}/launcher/benchmark-launcher-#{version.csv.first}-macos.dmg"
   name "Blender Open Data Benchmark"
   desc "3D performance benchmarking tool"
   homepage "https://opendata.blender.org/"
 
   livecheck do
-    url "https://opendata.blender.org/cdn/BlenderBenchmark#{version.major_minor}/launcher/"
-    regex(/href=.*benchmark[._-]launcher[._-](\d+(?:\.\d+)+)[._-]macos\.dmg/i)
+    url :homepage
+    strategy :page_match do |page|
+      match = page.match(%r{.*Benchmark(\d+(?:\.\d+)+)/.*benchmark[._-]launcher[._-](\d+(?:\.\d+)+)[._-]macos\.dmg}i)
+      next if match.blank?
+
+      "#{match[2]},#{match[1]}"
```

Comment:
```suggestion
    regex(%r{["'][^"']+?Benchmark(\d+(?:\.\d+)+)/[^"']*?benchmark[._-]launcher[._-](\d+(?:\.\d+)+)[._-]macos\.dmg}i)
    strategy :page_match do |page, regex|
      page.scan(regex).map { |match| "#{match[1]},#{match[0]}" }
```

This suggestion addresses a few areas:

* Use of `.*` should be strengthened to something contextually-appropriate, to reduce the risk of the regex matching something it shouldn't. The URL is found within JSON data in this instance, so we can restrict matching to the string literal using `[^"']+?`/`[^"']*?`.
* The regex should be established outside of the `strategy` block using `#regex` and passed in (i.e., not defined within the `strategy` block).
* The `strategy` block should use the `page.scan(regex).map { ... }` approach to format version strings. This will handle all matches on the page instead of only the first match. It's less of a concern in this specific case since there's currently only one available version on the homepage but it's better to standardize on this more flexible approach.